ena-submission/scripts/ena_submission_helper.py

```diff
@@ -129,6 +129,28 @@ def get_project_xml(project_set):
         "PROJECT": dataclass_to_xml(project_set, root_name="PROJECT_SET"),
     }
 
+def reformat_authors(authors: str) -> str:
+    """This function reformats the Loculus authors string to the format expected by ENA
+    Loculus format: `John A. Doe; Jane B. C. Roe;`
```

Comment:
```suggestion
    Loculus format: `Doe, John A.; Roe, Jane B. C.;`
```

This is the new loculus format output by prepro - I hadn't realized that when I wrote the function comment.